source/backend/api/Areas/Acquisition/Controllers/AcquisitionFileController.cs

```diff
@@ -239,7 +239,7 @@ public IActionResult GetFileCompensations(long id)
         /// <returns></returns>
         [HttpGet("{id:long}/comp-req-h120s")]
         [Produces("application/json")]
-        [ProducesResponseType(typeof(IEnumerable<H120CategoryModel>), 200)]
+        [ProducesResponseType(typeof(IEnumerable<CompensationFinancialModel>), 200)]
         [SwaggerOperation(Tags = new[] { "comp-req-h120s" })]
         public IActionResult GetFileCompReqH120(long id, bool? finalOnly)
```

Comment:
nit: Not necessarily a change for this pr, but 
GetFileCompReqH120 -> GetCompensationRequisitionFinancials